source/renderer/zettlr-body.js

```diff
@@ -657,39 +657,39 @@ class ZettlrBody {
       this._currentPopup = null
     }) // .makePersistent()
 
+    const searchForElement = document.getElementById('searchWhat')
+    const searchFor = () => searchForElement.value || ''
+    const replaceWithElement = document.getElementById('replaceWhat')
+    const replaceWith = () => replaceWithElement.value
     // If a regular expression was restored to the find popup, make sure to set
     // the respective class.
-    if (regexRE.test($('#searchWhat').val())) {
-      $('#searchWhat').addClass('regexp')
-      $('#replaceWhat').addClass('regexp')
+    if (regexRE.test(searchFor())) {
+      searchForElement.addClass('regexp')
+      replaceWithElement.addClass('regexp')
```

Comment:
Doesn't it need to be `searchForElement.classList.add('regexp')`? I couldn't find a vanilla `addClass`-function in the JS docs?

Reply:
Good catch!